scripts/type-generation/generate-typings-utils.js

```diff
@@ -0,0 +1,73 @@
+
+const changeType = (text) => {
+    if (text.indexOf('Array.') > -1) {
+        const arrayText = 'Array.<'
+        text = text.substring(arrayText.length, text.length - 1) + '[]'
+    }
+
+    switch (text) {
+    case 'Buffer':
+    case 'Function':
+    case 'RegExp': {
+        break
+    }
+    case 'Element': {
+        text = 'Element<void>'
+        break
+    }
+    case 'Element[]': {
+        text = 'Element<void>[]'
+        break
+    }
+    default: {
+        text = text.toLowerCase()
+    }
+
+    }
+    return text
+}
+
+const getTypes = (types, alwaysType) => {
+    types.forEach((type, index, array) => {
+        array[index] = changeType(type)
+    })
+    types = types.join(' | ')
+    if (types === '' && !alwaysType) {
+        types = 'undefined'
+    } else if (types === '*' || (types === '' && alwaysType)) {
+        types = 'any'
+    }
+
+    return types
+}
+
+const buildCommand = (commandName, commandTags, indentation = 0) => {
```

Comment:
Moved from webdriverio-generate-typings.js, added indentation param and removing default values from param names